texttospeech/ssml_addresses/tts.py

```diff
@@ -0,0 +1,123 @@
+"""Source code for SSML Tutorial for Cloud Text-to-Speech API."""
```

Comment:
Please add the license to the top of each file.
```
# Copyright 2019 Google LLC
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#      http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```

Reply:
Is it alright if I just add them to the python files?